index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <span>Hello, Microverse</span>
```

Comment:
You did an overall great job building your index.html file 💯. However, please make sure that you wrap your 'Hello, Microverse' using an `h1` tag instead of a `span` tag. Using the right semantic HTML tags is a good practice. Quote to the related requirement below. 👇🏽
  >A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.

Reply:
Thanks a lot @jmonto55, I will work on then immediately.